microsoft-edge/toc.yml

```
@@ -459,12 +459,22 @@
               href: webview2/get-started/wpf.md
         - name: Sample code
           href: webview2/code-samples-links.md
-        - name: Concepts
+        - name: Concepts and How-To
```

```suggestion
        - name: Concepts and How-to
```
Or, we could go with just "How to", like done in PWA: https://review.docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/?branch=pr-en-us-1606
The defined Content Types are designed for Azure services docs (using highly structured templates & TOC parent node labels for them), rather than for product docs, so there's only loose guidance (if any) from Docs Contributor Guide, for this TOC bucket label for this repo.